db/system_distributed_keyspace.cc

```diff
@@ -26,8 +26,12 @@
 #include "db/consistency_level_type.hh"
 #include "db/system_keyspace.hh"
 #include "schema_builder.hh"
+#include "service/storage_service.hh"
```

Comment:
This is a circular dependency, no? Because storage_service depends on everything.

Reply:
Removed